vercel/turbo
# chore(packages/ui turbo gen config.ts): Preventing the format of pack…

…age.json from being corrupted

### Description

Prevent the format of `package.json` from getting corrupted when creating a component with `turbo gen`.

Here's what the exports section looks like before and after the fix:

before

```
  "exports": {
"./foo": "./src/foo.tsx",
    "./button": "./src/button.tsx",
    "./card": "./src/card.tsx",
    "./code": "./src/code.tsx"
  },
```

after

```
  "exports": {
    "./foo": "./src/foo.tsx",
    "./button": "./src/button.tsx",
    "./card": "./src/card.tsx",
    "./code": "./src/code.tsx"
  },
```

### Testing Instructions

1. Create a component with `turbo gen`.
2. Check package.json.
